src/feditest/protocols/webfinger/__init__.py

```diff
@@ -90,35 +90,31 @@ class WebFingerClient(WebClient):
     """
     A Node that acts as a WebFinger client.
     """
-    def perform_webfinger_query(self, resource_uri: str, rels: list[str] | None = None) -> WebFingerQueryResponse:
+    def perform_webfinger_query(
+        self,
+        server: WebFingerServer,
+        resource_uri: str,
+        rels: list[str] | None = None,
+        scheme: str = 'https'
```

Comment:
Is this used somewhere? I looked at the feditest-tests-fediverse changes I didn't see any usage (but I could have easily missed something).

Reply:
I'm missing something. perform_webfinger_query is used all over the place in the WebFinger tests. This declaration just needs to have the same signature as the signature of the same method in the Imp subclass.

Reply:
I was asking about the new method argument rather than the method itself.

Reply:
Ah. This was to make it consistent with `construct_webfinger_uri_for`